spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/RuntimeAppsController.java

```diff
@@ -81,6 +84,8 @@ public int compare(AppInstanceStatus i1, AppInstanceStatus i2) {
 
 	private final ResourceAssembler<AppStatus, AppStatusResource> statusAssembler = new Assembler();
 
+	private ForkJoinPool forkJoinPool = new ForkJoinPool(8);
```

Comment:
Updated the PR.
For the record, 8 is the empirical value I came to when testing on a backend that has ~1s roundtrip time (CF) with a non trivial amount of apps (20). conclusion is that the FJP is very reluctant to steal work on low number of tasks (but which are IO blocked). Hardly saw 6 working at the same time. So 8 seemed like a good compromise.

Anyway, made that a bean in latest commit